docs/packaging/updating-an-existing-package.md

```diff
@@ -167,6 +167,20 @@ Full release notes:
 - [ ] Package was built and tested against unstable
 ```
 
+### Other commit message format examples
+
+In the cases where you are not updating a package to a new version, but simply applying a patch of fixing an existing version, the following commit message shortlogs are considered suitable:
+
+- Add the `homepage` metatdata key:
+  `package: Add homepage`
+- Fix a packaging issue (including adding a patch to fix the package)
+  `package: Fix (...)`
+- Updating a README, monitoring.yml file or other stuff that does not directly _change_ the package, meaning the package release number didn't need a bump:
+  `package: [NFC] Add README.md and monitoring.yml`
+`[NFC]` is an abbreviation adopted from the clang compiler suite, which expands to "Non Functional Change".
```

Comment:
I don't think history of the term is relevant here. It also adds unnecessary jargon. Please remove.

Reply:
When we introduced it in serpent, people asked "what does NFC stand for?!", because it is a common abbreviation for "No F...ing Clue", hence I added it for context.

Reply:
Okay, how about swap the senteces. Definition comes first then the context.